src/testsets.jl

```diff
@@ -102,6 +175,35 @@ function finish(ts::TestRelTestSet)
     return ts
 end
 
+function get_log_header(ts::TestRelTestSet, duration, database, engine_name)
+    io, ctx = get_logging_io()
+
+    # status
+    anyerror(ts) && write(ctx, "[ERROR]")
+    anyfail(ts) && write(ctx, "[FAIL]")
+    all_pass = !anyerror(ts) && !anyfail(ts)
+    all_pass && write(ctx, "[PASS]")
+
+    # core info
+    name = ts.dts.description
+    write(ctx, " $name, duration=$duration")
+
+    # tail
+    if all_pass
+        txnids = Set()
+        for log in ts.logs
+            if haskey(log.kwargs, :transaction_id)
+                push!(txnids, log.kwargs[:transaction_id])
+            end
+        end
+        write(ctx, """ TxIDs=[$(join(txnids, ", "))]""")
```

Comment:
why `TxIDs` not `TxnIDs`? (this isn't my world, do idk what standard abbreviations are, so i'm mostly just this isn't a typo)